hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip993/SystemFileExportsTest.java

```diff
@@ -457,11 +454,6 @@ private static VisibleItemsValidator nodeDetailsExportValidator(
                             actualCertHash,
                             "node" + address.nodeId() + " has wrong cert hash");
 
-                    final var expectedAccountID = AccountID.newBuilder()
-                            .accountNum(address.nodeId() + ACCOUNT_ID_OFFSET)
-                            .build();
-                    assertEquals(expectedAccountID, address.nodeAccountId());
-
```

Comment:
Why aren't these assertions needed anymore?

Reply:
The test was doing node AccountId update after prepare upgrade is scheduled. But after upgrade the state is same as the one in prepare upgrade. This is not possible unless we maintain candidate roster, active roster and future roster in state. Those are future items. 
Since node accountId update is disabled by default, I removed that in the test.